src/LaunchDarkly.ServerSdk/Files/FileDataSource.cs

```diff
@@ -98,6 +98,50 @@ private void LoadAll()
             _loadedValidData = true;
         }
 
+        private const int ReadFileRetryDelay = 200;
+        private const int ReadFileRetryAttempts = 30000 / ReadFileRetryDelay;
+

```

Comment:
Switched this to retry immediately the first time, but then wait 200ms there after until approximately 30 seconds is reached.  I don't really intend for the 30 seconds to ever be reached, but failure is devastating if the features don't get picked up.

Reply:
What's the purpose of retrying immediately the first time instead of using the same delay?

Reply:
In my testing the 2nd time has always succeeded.  That's with 914 KB on decent SSDs.  Can't promise everyone is going to see the same.  I do agree with you however that moving files around is the safer solution.  The retry is just nice for the human playing around.

Reply:
OK, no big deal either way.